mirage/psf/psf_selection.py

```diff
@@ -287,7 +287,10 @@ def get_library_file(instrument, detector, filt, pupil, wfe, wfe_group,
 
             # Compare the header entries to the user input
             file_inst = header['INSTRUME'].upper()
-            file_det = header['DETECTOR'].upper()
+            try:
+                file_det = header['DETECTOR'].upper()
+            except KeyError:
+                file_det = header['DET_NAME'].upper()
```

Comment:
Hmm, DET_NAME vs DETECTOR is probably our fault from writing inconsistent header keywords in different parts of WebbPSF. What keyword will DMS use for this? We can make sure WebbPSF is consistent from now on.

Reply:
DMS uses `DETECTOR`